en/components/grid-row-filter.md

```diff
@@ -18,7 +18,7 @@ In order to indicate that a Grid is filterable via a Row Filter, add a second He
 
 ## Filter State
 
-The Row Filter Header Cells come with a `Filter State` override in Sketch that lets you configure whether a filtering condition has been applied on a given column. By default, it shows **Empty**, but you may set it to Filtered and specify conditions via the [Chip](chips.md) it contains. In Adobe XD things are organized in a similar fashion via the  `Filter State` layer that uses the `Component States` paradigm to let you easily switch between states.
+The Row Filter Header Cells come with a `Filter State` override in Sketch that lets you configure whether a filtering condition has been applied on a given column. By default, it shows **Empty**, but you may set it to **Filtered** and specify conditions via the [Chip](chips.md) it contains. In Figma the Row Filtering is one of the Grid Header Cell types, that can be switched from the properties panel. By default, the filtering is set to **Empty**, but you can easily switch it to **Filtered** via the `Type` property. In Adobe XD things are organized in a similar fashion via the  `Filter State` layer that uses the `Component States` paradigm to let you easily switch between states.
```

Comment:
The description of row filtering is inaccurate. There's no "default" type of header cell and the types are called "Row Filter Filtered" and "Row Filter Empty".